todo_app/data/trello_items.py

```diff
@@ -0,0 +1,80 @@
+import json
+import os
+import requests
+
+KEY = os.getenv("TRELLO_API_KEY")
+TOKEN = os.getenv("TRELLO_API_TOKEN")
+BOARD_ID = os.getenv("TRELLO_BOARD_ID")
+NOT_STARTED_COLUMN = os.getenv("TRELLO_NOT_STARTED_LIST_ID")
+FINISHED_COLUMN = os.getenv("TRELLO_FINISHED_LIST_ID")
+TRELLO_API = "https://api.trello.com/1"
+
+class Item:
+  def __init__(self, id, name, status = "Not Started"):
+    self.id = id
+    self.name = name
+    self.status = status
+  
+  def from_trello_card(cls, card):
+    return cls(card['id'], card['name'],  ("Completed" if card["idList"] == FINISHED_COLUMN else "Not Started"))
+
+BASE_QUERY = {
+  "key": KEY,
+  "token": TOKEN
+}
+
+HEADERS = {
+  "Accept": "application/json"
+}
+
+def add_item(title):
+  url = f"{TRELLO_API}/cards"
+
+  query = BASE_QUERY | {
+    "idList": "65cf68e717e5358c95f22842",
```

Comment:
We should switch this for the environment variable so we're not locked into this Trello board